sys/kern/timer.c

```diff
@@ -9,6 +9,7 @@
 static mtx_t timers_mtx = MTX_INITIALIZER(0);
 static timer_list_t timers = TAILQ_HEAD_INITIALIZER(timers);
 static timer_t *time_source = NULL;
+static bintime_t timebasebin = BINTIME(0);
```

Comment:
Strange choice for naming. Shouldn't it be `basebintime`?

Reply:
Agree, but in my opinion, `boottime` will be the best option.